src/style.css

```diff
@@ -0,0 +1,159 @@
+:root {
+  --main-accent: #060b35;
+  --secondary-color: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #e5e5e5;
+  --yellow-stars-color: #ffde6a;
+  --card-padding: 16px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-accent);
+}
+
+body,
+div,
+p {
+  margin: 0;
+}
+
+.card {
+  max-width: calc(198px - (var(--card-padding) * 2));
+  max-height: 400px;
+  padding: var(--card-padding);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-color: var(--white);
+
+  font-weight: 500;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image-iMac {
+  display: flex;
+
+  margin: 16px 0 40px;
+  width: 160px;
+  height: 134px;
+
+  background-image: url(images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.card__title {
+  display: flex;
+  align-items: center;
+
+  margin-bottom: 4px;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__code {
+  display: flex;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__star {
+  display: flex;
+}
+
+.card__stars-star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__stars--1 :nth-child(1),
+.card__stars--2 :nth-child(-n+2),
+.card__stars--3 :nth-child(-n+3),
+.card__stars--4 :nth-child(-n+4),
+.card__stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__review {
+  display: flex;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.card__price-list {
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  padding: 0;
+  margin-bottom: 16px;
+
+  list-style: none;
+}
+
+.card__price-title {
+  display: flex;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.card__price-amount {
+  display: flex;
+  text-align: center;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy-button {
+  border: 1px solid var(--blue-accent);
+  border-radius: 5px;
+  background-color: var(--blue-accent);
+}
+
+.card__buy-button-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 38px;
+
+  font-size: 14px;
+  font-weight: 700;
+  color: var(--white);
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.card__buy-button:hover {
+  background-color: var(--white);
+}
+
+.card__buy-button-link:hover {
+  color: var(--blue-accent);
+}
```

Comment:
use only one :hover pseudo-class after merging styles to `<a`

Reply:
border-radius didn't work then